src/index.html

```diff
@@ -22,6 +22,426 @@
+          class="card__link"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <div class="card__image">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac Product Image"
+            class="card__image"
+          />
+        </div>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviev">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-tag">Price:</p>
+
+          <p class="card__price-price">$2,199</p>
+        </div>
+
+        <a
+          class="card__link"
+          data-qa="hover"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <div class="card__image">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac Product Image"
+            class="card__image"
+          />
+        </div>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviev">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-tag">Price:</p>
+
+          <p class="card__price-price">$2,199</p>
+        </div>
+
+        <a
+          class="card__link"
+          data-qa="hover"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <div class="card__image">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac Product Image"
+            class="card__image"
+          />
+        </div>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviev">Reviews: 5</p>
```

Comment:
There is a typo in your class name. It should be 'card__review' instead of 'card__reviev'.